src/Log/LogRepository.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+namespace Give\Log;
+
+/**
+ * Class LogRepository
+ * @package Give\Log\Repositories
+ *
+ * @since 2.9.7
+ */
+class LogRepository {
```

Comment:
This repository could really benefit from a `Log` model. Right now an associative array is being passed around.

Reply:
I agree. 
Resolved 0db0fc60